alerts/machine-config-operator/MachineConfigDaemonPivotError.md

```diff
@@ -0,0 +1,151 @@
+# MCDPivotError
+
+## Meaning
+
+Alerts the User when the Machine Config Daemon
+is either unable to or detecs an error when
+trying to pivot OS image to a new version.
+
+>Alerts the user when an error is detected upon pivot.
+This triggers if the pivot errors are above zero for 2 minutes.
+
+This alert occurs when a node experiences
+OS update or kernel change failure(s)
+lasting over 2 minutes during a cluster upgrade.
+
+## Impact
+
+If the MCD is unable to update the OS and
+finish pivoting then this can prevent an
+OpenShift upgrade from completing.
+This can leave the cluster an a
+unstable state and further affect the operation of the cluster.
+
+## Diagnosis
+
+When a node pivots to update the OS image.
+This is done via the `rpm-ostree` service
+that logs any actions taken to the `machine-config-daemon-*` pod
```

Comment:
Line 26 is an incomplete sentence. The following is my attempt to complete the sentence:
```suggestion
When a node pivots to update the OS image,
the `rpm-ostree` service logs any actions taken to the `machine-config-daemon-*` pod.
```
or 
```suggestion
When a node pivots to update the OS image,
the `rpm-ostree` service logs any actions performed on the `machine-config-daemon-*` pod.
```

Reply:
Grammar fixed.